### ISSUE TITLE
v13 `Option::unwrap()` on a `None` value and assertion failed: !head_indices.is_empty()

### ISSUE THREAD

**Issue by hudja** [PacificBiosciences/pb-StarPhase#29]
Hello, I have a problem with couple of samples using the latest v13 executable and db.

```
[W::hts_idx_load3] The index file is older than the data file: lq/hiphase/zzz/zzz.aligned.haplotagged.bam.bai
[2024-08-13T10:49:51.646Z ERROR pbstarphase::cyp2d6::caller] Error while typing consensus #0, setting to UNKNOWN.
[2024-08-13T10:49:51.646Z ERROR pbstarphase::cyp2d6::caller] Typing error: no matches found
[2024-08-13T10:49:51.712Z ERROR pbstarphase::cyp2d6::caller] Error while typing consensus #0, setting to UNKNOWN.
[2024-08-13T10:49:51.712Z ERROR pbstarphase::cyp2d6::caller] Typing error: no matches found
[2024-08-13T10:49:51.728Z INFO  pbstarphase::cyp2d6::caller] Generating CYP2D6 BAM at "lq/pbstarphase/zzz/zzz.cyp2d6.bam"
[2024-08-13T10:49:51.761Z INFO  pbstarphase::cyp2d6::caller] Finished building index for "lq/pbstarphase/zzz/zzz.cyp2d6.bam"
[2024-08-13T10:49:51.776Z WARN  pbstarphase::cyp2d6::caller] 	Removed putative allele mid-read due to no good matches, chaining may be impacted.
thread 'main' panicked at src/cyp2d6/chaining.rs:544:43:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

and

```
[2024-08-13T10:54:26.286Z INFO  pbstarphase::cyp2d6::caller] Solving CYP2D6...
[W::hts_idx_load3] The index file is older than the data file: lq/hiphase/zzz/zzz.aligned.haplotagged.bam.bai
[2024-08-13T10:54:27.624Z INFO  pbstarphase::cyp2d6::caller] Generating CYP2D6 BAM at "lq/pbstarphase/zzz/zzz.cyp2d6.bam"
[2024-08-13T10:54:27.647Z INFO  pbstarphase::cyp2d6::caller] Finished building index for "lq/pbstarphase/zzz/zzz.cyp2d6.bam"
thread 'main' panicked at src/cyp2d6/chaining.rs:294:5:
assertion failed: !head_indices.is_empty()
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Not sure what would be the best way to proceed with debugging.

Best wishes,
Georgi

**Comment by holtjma** [comment]
Received some additional information via email communications. Coverage is likely an issue in both datasets, but we should patch the software so that there is not a crash and instead some informative error message.

**Comment by holtjma** [comment]
This was resolved with updates in v0.13.3, full release available here: 

https://github.com/PacificBiosciences/pb-StarPhase/releases/tag/v0.13.3
